index.js

```diff
@@ -99,4 +99,17 @@ module.exports = class BemEntityName {
     isEqual(entity) {
         return entity && (this.id === entity.id);
     }
+
+    /**
+     * Determines whether specified entity belongs to this.
+     * @param {object} entity - the entity to compare.
+     * @returns {boolean}
+     */
+    belongsTo(entity) {
```

Comment:
`({block: 'a'}).contains({block: 'a', elem: 'e'})` ? What does this mean? It's not true even in DOM.